west.yml

```diff
@@ -199,7 +199,7 @@ manifest:
       groups:
         - hal
     - name: hal_nxp
-      revision: c42d70dba969de09c0e43d3b26a3ffcb015f6f33
+      revision: pull/434/head
```

Comment:
Why is HAL update made in the commit adding board support? Are the HAL changes only needed once the board is enabled?

Reply:
I think this HAL update should be included in this commit since it is being enabled the flash node. Without the correct HAL update, the build will fail in this commit.

Reply:
I updated to move this into the proper commit